Prox/Prox/Models/PlaceProviders.swift

```diff
@@ -42,11 +42,14 @@ class SinglePlaceProvider: PlaceProvider {
             self.description = nil
         }
 
-        if let categoriesFromFirebase = dict["categories"] as? [[String:String]] {
-            categories = SinglePlaceProvider.getCategories(fromFirebaseValue: categoriesFromFirebase)
-        } else {
-            categories = ([], [])
+        let categoryIds = dict["categories"] as? [String] ?? []
+        var categories = (names: [String], ids: [String])([], [])
```

Comment:
super nit: no need to waste time changing now but consider for next time. I think this would be clearer:
```swift
var categories: (names: [String], ids: [String]) = ([], [])
```